parktain/main.py

```diff
@@ -65,12 +65,23 @@ def checkins_reminder():
     date = datetime.now().strftime('%d %B, %Y')
     bot.speak('Morning! What are you doing on {}!'.format(date), "#checkins")
 
-@bot.listen_for(lambda user, channel, message: True)
+@bot.listen_for(lambda user, channel, message: True, target_channel='clickbaits')
 def logger(user, channel, message):
     message_log = Message(user_id=user, channel_id=channel, message=message, timestamp=datetime.now())
     session.add(message_log)
     session.commit()
 
+    # Check for presence of HyperLink in message
+    if user == bot.id:
```

Comment:
I pushed a change to `gendo` to do this.  Or do you think some handlers should capture even bot messages? For instance the logger itself?